src/main/java/com/esri/samples/map/read_geopackage/ReadGeoPackageSample.java

```diff
@@ -68,108 +55,33 @@ public void start(Stage stage) {
-      Callback<ListView<Layer>, ListCell<Layer>> cellFactory = list -> new ListCell<Layer>() {
-
-        @Override
-        protected void updateItem(Layer layer, boolean bln) {
-
-          super.updateItem(layer, bln);
-          if (layer != null) {
-            if (layer instanceof FeatureLayer) {
-              FeatureLayer featureLayer = (FeatureLayer) layer;
-              setText(featureLayer.getName());
-            } else if (layer instanceof RasterLayer) {
-              RasterLayer rasterLayer = (RasterLayer) layer;
-              // use the raster file name if the raster layer name is empty
-              String path = rasterLayer.getRaster().getPath();
-              setText(rasterLayer.getName().isEmpty() ? path.substring(path.lastIndexOf('/') + 1) : rasterLayer.getName());
-            }
-          } else {
-            setText(null);
-          }
-        }
-
-      };
-
-      mapLayers.setCellFactory(cellFactory);
-      geoPackageLayers.setCellFactory(cellFactory);
-
-      // when you click on a layer in the geopackage layers list view, add it to the map
-      geoPackageLayers.setOnMouseClicked(e -> {
-        if (e.isStillSincePress() && e.getButton() == MouseButton.PRIMARY) {
-          // get selected layer
-          Layer layer = geoPackageLayers.getSelectionModel().getSelectedItem();
-          if (layer != null) {
-            // add it to the map and the top of the map layers list
-            map.getOperationalLayers().add(layer);
-            mapLayers.getItems().add(0, layer);
-            // remove it from the geoPackage layers list
-            geoPackageLayers.getItems().remove(layer);
-          }
-        }
-      });
-
-      // when you click on a layer in the map layers list view, remove it from the map
-      mapLayers.setOnMouseClicked(e -> {
-        if (e.isStillSincePress() && e.getButton() == MouseButton.PRIMARY) {
-          // get selected layer
-          Layer layer = mapLayers.getSelectionModel().getSelectedItem();
-          if (layer != null) {
-            // remove it from the map and the map layers list
-            map.getOperationalLayers().remove(layer);
-            mapLayers.getItems().remove(layer);
-            // add it back to the geoPackage layers list
-            geoPackageLayers.getItems().add(layer);
-          }
-        }
-      });
-
-      // read the raster and feature layers from the geoPackage and show them in the list view
+      // open the Geopackage
       File geoPackageFile = new File("./samples-data/auroraCO/AuroraCO.gpkg");
       GeoPackage geoPackage = new GeoPackage(geoPackageFile.getAbsolutePath());
       geoPackage.loadAsync();
       geoPackage.addDoneLoadingListener(() -> {
+
+        // get the Rasters from the GeoPackage, and add them to the map as a layer
         geoPackage.getGeoPackageRasters().forEach(raster -> {
           RasterLayer rasterLayer = new RasterLayer(raster);
           rasterLayer.loadAsync();
```

Comment:
as the geopackage is already loaded asynchronously, it's not necessary to call `.loadAsync` on each raster layer.